strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java

```diff
@@ -53,6 +55,8 @@ protected void configure(HttpSecurity http)
         http.sessionManagement()
             .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
             .and()
+            .exceptionHandling().authenticationEntryPoint(basicAuthenticationEntryPoint())
```

Comment:
@fuss86 if you agree with sugestion then it would be great to add a `TODO` about this.